blue_level/pizzabot.js

```diff
@@ -0,0 +1,54 @@
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
+
+const validateOrderName = (orderName) => {
+  if (
+    orderName === "vegetarian" ||
+    orderName === "pepperoni" ||
+    orderName === "hawaiian"
+  ) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Great job :)
Regarding clean code, there is also another convention, let me introduce you to it.
Apart from returning given thing from function, `return` keyword does one more thing - it automatically finishes function. That's why we mustn't write anything below `return`, because that code will never be executed (of course in your case we have code below `return true`, but it's inside if statement, so it wouldn't enter second statement in the first place).
Thanks to that feature of `return` keyword, we can write our function in slightly different, far more clean fashion :

```
const validateOrderName = (orderName) => {
  if (orderName === "vegetarian" || orderName === "pepperoni" || orderName === "hawaiian") {
    return true;
  }
  return false;
}
```

As you can see, we are not writing `else` statement anymore. We can do that thanks to behaviour mentioned above; if result of if statement is `true`, we will execute `return true` code from line 12 and finish function. If, on the other hand, result of it is `false`, we will never enter line 12 and jump straight into line 14. After executing line 14 we will finish function same way :)

Reply:
And again! Thank you for showing us how to write cleaner code.